### ISSUE TITLE
Fails on GAE because of "unsafe" import

### ISSUE THREAD

**Issue by etnbrd** [valyala/fasttemplate#3]
`fasttemplate` is used by echo, a web framework, for its logger.
I currently use echo on Google App Engine, and it throws the following error :
```
2017/01/16 16:08:42 go-app-builder: Failed parsing input: parser: bad import "unsafe" in github.com/valyala/fasttemplate/template.go from GOPATH
```
As far as I understand, `fasttemplate` now relies heavily on `unsafe` for performance optimizations.
How could we workaround this so that `fasttemplate` and `echo` stay compatible with GAE ?

**Comment by ghost** [comment]
@valyala having the same issue.

**Comment by dhanunjay** [comment]
@valyala I am also hit by this bug on GAE. is there any workaround?

**Comment by ramesh-lingappan** [comment]
guys, any update on this?

**Comment by ghost** [comment]
@valyala could this PR be merged? It's been open for quite a while now.
This fix is urgently needed

**Comment by nigeon** [comment]
@valyala having the same issue. Please could the PR be merged?
Thanks.

**Comment by valyala** [comment]
Merged the PR #4 that fixes the issue.
